internal/service/directconnect/vif.go

```diff
@@ -28,12 +28,22 @@ func dxVirtualInterfaceRead(id string, conn *directconnect.DirectConnect) (*dire
 func dxVirtualInterfaceUpdate(d *schema.ResourceData, meta interface{}) error {
 	conn := meta.(*conns.AWSClient).DirectConnectConn
 
-	if d.HasChange("mtu") {
+	if d.HasChanges("mtu") {
```

Comment:
If you are making separate calls to [`UpdateVirtualInterfaceAttributes`](https://docs.aws.amazon.com/directconnect/latest/APIReference/API_UpdateVirtualInterfaceAttributes.html) then please keep `d.HasChange()`.